core/templates/components/state-editor/state-editor.component.spec.ts

```diff
@@ -110,8 +109,7 @@ describe('State Editor Component', () => {
       new SubtitledHtml('<p> Previous HTML string </p>', 'Id'),
     ));
     component.sendOnSaveNextContentIdIndex(0);
-    component.sendOnSaveInteractionCustomizationArgs('');
-    component.sendOnSaveInteractionId('');
+    component.sendOnSaveInteractionData('');
```

Comment:
Is the argument here correct? I didn't think interaction data would be a string...

If it's an error, why did the test pass / why did no checks detect this?